downstreamadapter/dispatcher/dispatcher.go

```diff
@@ -451,9 +452,8 @@ func (d *Dispatcher) PassBlockEventToSink(event commonEvent.BlockEvent) {
 }
 
 func isCompleteSpan(tableSpan *heartbeatpb.TableSpan) bool {
```

Comment:
can we use common.IsCompleteSpan directly

Reply:
In fact, there are two `IsCompleteSpans` in dispatcher and coordinator. However, the implementation of these two functions is different.